External/SPMmodified/xASL/xASL_spm_smooth.m

```diff
@@ -117,15 +117,20 @@ function xASL_spm_smooth(pathIn, fwhmSmooth, pathNew)
 % ------------------------------------------------------------------------------------------
 % Rename temporary SPM file into output file
 % ------------------------------------------------------------------------------------------
+
 if ~isempty(pathNew) % otherwise don't rename, keep prefix "s" for smoothed
 
     TempName = fullfile(Fpath, ['s' Ffile Fext]);
-
+    
     if ~strcmp(pathNew,TempName)
-        pathDest=fullfile(Fpath, pathNew);
-        xASL_Move(TempName,pathDest,1);
+        [FpathN, ~, ~] = xASL_fileparts(pathNew);
+        if isempty(FpathN) % This means that the user didn't choose a fullpath, only a file name, so FpathN = ' '.
+            pathDest=fullfile(Fpath, pathNew);
+            xASL_Move(TempName,pathDest,1);
```

Comment:
I would insert `xASL_adm_CreateDir(Fpath);` here to be sure. This function creates the Fpath folder only if it doesn't exist!